lib/api/helpers.rb

```diff
@@ -55,6 +55,22 @@ def find_project(id)
       end
     end
 
+    def find_group(id)
+      begin
+        group = Group.find(id)
+      rescue ActiveRecord::RecordNotFound
+        group ||= Group.find_by(path: id)
```

Comment:
If you make the `Group.find_by` a bang method like so: `Group.find_by!(path: id)` it will throw a `ActiveRecord::RecordNotFound`. That way you dont need the code on line 65